CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -82,36 +82,61 @@ class EditingViewController: UIViewController {
     private let tableView = UITableView(frame: .zero, style: .grouped)
 
     // MARK: - Display Settings
-    var showsGroups = true
-    private var showingLess = true
-    private var hideAfter = 3 // Doesn't display more [hideAfter] categories if showingLess is true
+    private var showsGroups = true
+    private var showingLessRows = true
+    private var numberHiddenShownRows = 3 // Displays only [numberHiddenShownRows] if showingLessRows is true
 
     private var sections: [Section] = []
+
     // More Section and count
     private var moreSection: Section? {
         get {
             if let loc = sections.firstIndex(where: { $0.type == .more }) {
                 return sections[loc]
-            } else { return nil }
+            } else {
+                return nil
+            }
         }
     }
-    private var moreCount: Int {
-        get {
-            return moreSection?.filteredItems.count ?? 0
-        }
+    private var moreSectionSize: Int {
+        get { return moreSection?.filteredItems.count ?? 0 }
```

Comment:
With Swift 5 I think we should have implicit return, no? Also, it's a little inconsistent to expand `else { return nil }` but to inline `get { return ... }`.